tests/integration/test_integration.py

```diff
@@ -26,6 +29,26 @@
 @pytest.mark.usefixtures("migrate_db")
 class IntegrationTestCase(common.BaseTestCase):
 
+    def setUp(self):
```

Comment:
Do we still need these changes now that we have `test_candidates`?

Reply:
These are not needed--just removed! thanks @lbeaufort !